encryption/src/main/java/com/diyasylum/encryption/SecretBox.java

```diff
@@ -47,9 +48,16 @@ public SecretBox(String password) {
   private static boolean isInitialized = false;
 
   private static void loadLibrary() {
-    String libraryPath = "/usr/local/lib/libsodium.so";
+    String libraryPath;
+    if (Platform.isMac()) {
```

Comment:
This was from the docs of the libsodium jna library. I wanted to have a green build locally even if we reasonably expect this to only run on linux machines in production